Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/LambdaFunctionTemplate.tt

```diff
@@ -388,5 +389,28 @@ namespace <#= _model.LambdaMethod.ContainingNamespace #>
     }
 #>
         }
+
+        private static void SetExecutionEnvironment()
+        {
+            const string envName = "AWS_EXECUTION_ENV";
+            const string amazonLambdaAnnotations = "amazon-lambda-annotations";
+
+            var assemblyVersion = typeof(LambdaFunctionAttribute).Assembly
```

Comment:
We shouldn't need to do reflection for this. The source generator should be able to determine the version at compile time and put the value there.

Reply:
Fixed